src/CoreWCF.CrankTests/CoreWCFPerfService/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using CoreWCF;
+using CoreWCF.Configuration;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+
+namespace CoreWCFPerfService
+{
+    public class Parameters
+    {
+        public const string Url = "baseaddress";
+    }
+
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            string filePath = Path.Combine(Environment.CurrentDirectory, "CoreWCF.PerfService.exe");
+            Console.WriteLine(filePath);
+            string command = $" advfirewall firewall add rule name=\"CoreWCFPerfService\" dir=in protocol=any action=allow program=\"{filePath}\" enable=yes";
+            ExecuteCommand(command, Environment.CurrentDirectory, TimeSpan.FromSeconds(20));
+            Console.WriteLine("Application start.");
+
+            using (var host = CreateWebHostBuilder(args).Build())
+            {
+                host.Start();
+
+                Console.WriteLine("Service is Ready");
+                Console.WriteLine("Press Any Key to Terminate...");
+                Console.ReadLine();
+            }
+
+            command = $" advfirewall firewall delete rule name=\"CoreWCFPerfService\" program=\"{filePath}\"";
+            ExecuteCommand(command, Environment.CurrentDirectory, TimeSpan.FromSeconds(20));
+            Console.WriteLine("Clean up the firewall rule.");
+        }
+
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+            .UseKestrel(options => { options.ListenLocalhost(8080); })
+            .UseNetTcp(8808)
```

Comment:
Probably don't want to use ListenLocalhost as that won't accept requests from other hosts. You probably want to listen on IPv6Any